circle.yml

```diff
@@ -11,8 +11,12 @@ dependencies:
     - curl -L -o google-chrome.deb https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
     - sudo dpkg -i google-chrome.deb
     - rm google-chrome.deb
+    - sudo apt-get update
+    - sudo apt-get update; sudo sudo apt-get install libgdal1h libgdal-dev gdal-bin libcairo2-dev libpango1.0-dev
```

Comment:
The list of packages to install here isn't exactly the same as what's in the README — do we want to just include this line in the README?